src/components/common/DeleteCollectionModal/DeleteCollectionModal.tsx

```diff
@@ -21,6 +25,7 @@ export class DeleteCollectionModal extends React.Component<Props, State> {
 
     this.state = {
       isLoading: false,
+      collections: null,
```

Comment:
Why do you keep collections on the state here? It seems like you're calling `requestCollections`, store the results on the state of the modal and then just close it. `requestCollections` will update the collections list in the context, so there should be no need to store them in the modal.